pl.lproj/Localizable.strings

```diff
@@ -0,0 +1,136 @@
+"missingAPIKey" = "Brak klucza API";
+"savedApiKey" = "Klucz API zapisany!";
+"errorModels" = "Wprowadź klucz API, aby wyświetlić modele";
+"readOnlyModels" = "Model jest konfigurowany przez serwer na podstawie dostępności.\n\nDomyślny model to gpt-3.5-turbo.";
+"errorNetwork" = "Nie można połączyć się z AppStore";
+"detailNetwork" = "Aplikacja nie może bezpiecznie połączyć się z AppStore.\n\nZalogowanie się do iCloud zwykle rozwiązuje ten problem.";
+"gptModel" = "Model GPT";
+"prompt" = "Prompt";
+"reply" = "Odpowiedź";
+"exportLog" = "Eksportuj dziennik";
+"promptSuggestions" = "Tematy";
+"me" = "Ja";
+"AKME" = "AKME";
+"updateApp" = "Twoja wersja aplikacji nie jest już kompatybilna.\nZaktualizuj do najnowszej wersji.";
+"insufficientTokens" = "Nie masz wystarczającej liczby tokenów";
+"unknownNetworkError" = "Wystąpił błąd sieciowy, spróbuj ponownie później";
+"titleLongThread" = "Długi wątek";
+"warningLongThread" = "Ten wątek staje się długi - zacznij nowy, aby uniknąć dodatkowych kosztów.";
+"errorLongThread" = "Ten wątek jest zbyt długi, zacznij nowy.";
+"blockLongThread" = "Ten wątek przekroczył limit tokenów, nie możesz wysyłać nowych wiadomości";
+"transactionStarted" = "Transakcja rozpoczęta";
+"transactionFailed" = "Transakcja nie powiodła się";
+"transactionRestored" = "Transakcja przywrócona";
+"transactionSuccesful" = "Transakcja zakończona sukcesem";
+"transactionDeferred" = "Transakcja odroczona";
+"waitingModeration" = "Oczekiwanie na moderację";
+"reviewMessage" = "Twoja wiadomość może naruszać nasze zasady moderacji, proszę przejrzeć wiadomość.";
+
+/*Pomoc*/
+"instructionsTitle" = "Instrukcje obsługi";
+"websiteTitle" = "FAQ";
+"feedbackTitle" = "Opinia";
+"instructionsDetail" = "Przed rozpoczęciem pracy, prosimy przeczytać poniższe ważne instrukcje, aby zapewnić najlepsze doświadczenie z AKME.";
+"websiteDetail" = "Więcej odpowiedzi na często zadawane pytania znajdziesz na naszej stronie internetowej.";
+"feedbackDetail" = "Jeśli napotkasz jakiekolwiek problemy lub masz sugestie dotyczące poprawy aplikacji, prosimy o kontakt z nami!";
+"instructionsAction" = "Przeczytaj instrukcje";
+"websiteAction" = "Otwórz stronę internetową";
+"feedbackAction" = "Wyślij opinię";
+"reportIssue" = "Zgłoś problem";
+"reviewApp" = "Oceń AKME";
+"mailFeedback" = "Opinia o AKME";
+"problemFeedback" = "Problem z AKME";
+"describeIssue" = "Proszę opisać problem, z którym się zmagasz.\nDołączono także dziennik ostatnich transakcji oraz kopię pliku konfiguracyjnego. Te pliki nie zawierają żadnych informacji identyfikujących.\nProszę nie udostępniać poufnych lub prywatnych informacji.";
+"writeFeedback" = "Bardzo chętnie wysłuchamy Twoich opinii, ale prosimy o nieudostępnianie poufnych lub prywatnych informacji.";
+"errorSendingMail" = "Nie można wysłać e-maila";
+"chooseWisely" = "Wybierz opcję";
+"error" = "Błąd";
+"networkError" = "Wystąpił błąd sieciowy, spróbuj ponownie";
+"retryRequest" = "Powtórzyć żądanie?";
```

Comment:
```suggestion
"retryRequest" = "Powtórzyć zapytanie?";
```
Zapytanie is more like asking for something. Żądanie is more like strong demand.